export/delimited-zip/README.MD

```
@@ -0,0 +1,61 @@
+`job:ready`
+
+<!-- END_INFOCARD -->
+
+
+> When embedding Flatfile, this plugin should be deployed in a server-side listener. [Learn more](/docs/orchestration/listeners#listener-types)
+
+
+
+## Parameters
+
+#### `job` - `string` 
```

Markdownlint

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)